server/app-engine/static/graas.js

```diff
@@ -348,11 +360,11 @@ function handleRefreshOkay() {
 function handleStartStop() {
     util.log("handleStartStop()");
 
-    var p = document.getElementById('start-stop');
+    var p = document.getElementById(START_STOP_BUTTON);
     var text = p.textContent || p.innerText;
     util.log("- text: " + text);
 
-    if (text == "Load trips") {
+    if (text == START_STOP_BUTTON_LOAD_TEXT) {
```

Comment:
js has `==` behavior that is unintuitive to me where it does type conversion. `===` is almost always what we want instead of the former.

Reply:
Thank you for catching this - clearly ==, ===, equals() etc  is an area I should focus on more, will do so. Updated, tested, will merge now.